examples/null.md

```diff
@@ -0,0 +1,141 @@
+# Null primitive type
+
+The `nullable` type attribute can be used to outline that `null` is an accepted value.
+
+## Using `null` with `string` and `array` types
+
+### MSON
+
+```mson
+- keyA (string, nullable, required) - Accepts `null` or `string`
+- keyB (array, nullable, required) - Accepts `null` or an `array`
+```
+
+### JSON
+
+```json
+{
+   "keyA": null,
+   "keyB": null
+}
+```
+
+```json
+{
+   "keyA": "bob",
+   "keyB": [1, 2, 3]
+}
+```
+
+## Using `null` as a possible value for an array
+
+### MSON
+
+```mson
+- keyA (array[enum], required) Accepts an array of `null` or `string`
+    - (string, required)
+```
```

Comment:
I was attempting to say that it requires a string input, so anything else including empty would be invalid. But re-reading the MSON spec that is not correct. 

This would be a more accurate example of an array that accepts string or null.

```mson
- keyA (array, required) Accepts an array of `null` or `string`
    - (string, nullable)
```